tests/utility/test_db_gateway.py

```diff
@@ -37,11 +37,11 @@ def setUpClass(cls):
         cls.ver = '1'
         cls.pkg = 'pkg'
         # Read Vendor Data from JSON.
-        gremlin_batch_data = os.path.join('/bayesian/tests/data/gremlin/gremlin_batch_data.json')
```

Comment:
Remove absolute path from tests enable to execute tests locally without building docker images